<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Phylogenetic trees</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marco Chiapello" />
    <meta name="date" content="2020-12-22" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <script src="libs/clipboard/clipboard.min.js"></script>
    <link href="libs/shareon/shareon.min.css" rel="stylesheet" />
    <script src="libs/shareon/shareon.min.js"></script>
    <link href="libs/xaringanExtra-shareagain/shareagain.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-shareagain/shareagain.js"></script>
    <link rel="stylesheet" href="scrollable.css" type="text/css" />
    <link rel="stylesheet" href="mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="fonts_mtheme_max.css" type="text/css" />
    <link rel="stylesheet" href="extra.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




class: title-slide

&lt;br&gt;
.font200[.f700[Phylogenetic trees]]&lt;br&gt;
.font120[Quick tutorial]

&lt;br&gt;
&lt;br&gt;
.marco[
Marco Chiapello
&lt;br&gt;
22 December, 2020
]

---
class: clear

&lt;br&gt;&lt;br&gt;
# .black[.font200[Agenda]]

- Common phylogeny terminology, concepts and tools

- An example tree

---

layout: true
# Common phylogeny terminology, concepts and tools

---

## What is a phylogenetic tree?

.par30[
A phylogenetic tree or evolutionary tree is a branching diagram or "tree" showing the evolutionary relationships among various biological species or other entities
]

---

&lt;img src="images/15.48.32.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.48.57.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Which of these topologies are the same?

&lt;img src="images/15.49.06.png" width="1000px" style="display: block; margin: auto;" /&gt;

---

## Rooting trees
&lt;img src="images/15.49.27.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Branch lengths
&lt;img src="images/15.50.12.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.50.22.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Common representations
&lt;img src="images/15.50.37.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.50.54.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.51.17.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Newick format

.par30[
- A string describing tree topology and branch lengths
- Popular input and output format for phylogenetic software
- Example: (A:1,D:6,(((E:1,F:1):1,B:2):1,(C:4,G:2):2):1);
&lt;br&gt;&lt;br&gt;
https://en.wikipedia.org/wiki/Newick_format
]

---

&lt;img src="images/15.54.35.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.54.44.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.54.56.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.01.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.04.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.13.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.20.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.25.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.36.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.41.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.45.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.55.54.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.04.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.09.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.13.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.19.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.23.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.28.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.33.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.38.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.45.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.56.55.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.00.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.12.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.17.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.22.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.29.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.41.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.46.png" width="800px" style="display: block; margin: auto;" /&gt;

---

&lt;img src="images/15.57.53.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Which sites are related?

&lt;img src="images/17.24.07.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Differences indicated by an alignment
&lt;img src="images/17.24.23.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Alignment for phylogenetics
&lt;img src="images/17.24.30.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Columns in alignments should be homologous
&lt;img src="images/17.25.18.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## There is only one true tree
&lt;img src="images/17.25.42.png" width="800px" style="display: block; margin: auto;" /&gt;

---

## Methodological approaches

.par30[
1. Distance matrix methods (pre-computed distances)
	- UPGMA	assumes perfect molecular clock Sokal &amp; Michener (1958) 
	- Minimum evolution (e.g. Neighbour-joining, NJ) Saitou &amp; Nei (1987)
1. Maximum parsimony Fitch (1971)
	- Minimises number of mutational steps 	
1. Maximum likelihood, ML
    - Evaluates statistical likelihood of alternative trees, based on an explicit model of substitution
1. Bayesian methods
    - Like ML but can incorporate prior knowledge
]

---

## Methodological approaches

.par30[

1. .opacity10[Distance matrix methods (pre-computed distances)]
	- .opacity10[UPGMA	assumes perfect molecular clock Sokal &amp; Michener (1958) ]
	- .opacity10[Minimum evolution (e.g. Neighbour-joining, NJ) Saitou &amp; Nei (1987)]
1. .opacity10[Maximum parsimony Fitch (1971)]
	- .opacity10[Minimises number of mutational steps]
1. Maximum likelihood, ML
    - Evaluates statistical likelihood of alternative trees, based on an explicit model of substitution
1. .opacity10[Bayesian methods ]
	- .opacity10[Like ML but can incorporate prior knowledge ]
]

---
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="libs/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9",
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
